spec/features/budgets/access_show.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+feature "views show page" do
```

Comment:
A suggestion to change file name and feature title to `"views budget"`

Reply:
Much better - done in sha: ec18df2